README.md

```diff
@@ -27,6 +27,25 @@ Install the latest version of `confluent` to `/usr/local/bin`:
 
     brew install confluentinc/tap/cli
 
+#### APT (Ubuntu and Debian)
+
+Install the latest version of `confluent` to `/usr/bin` (requires `glibc 2.17` or above for `amd64` and `glibc 2.27` or above for `arm64`):
+
+    wget -qO - https://packages.confluent.io/confluent-cli/deb/archive.key | sudo apt-key add -
+    sudo apt install software-properties-common
+    sudo add-apt-repository "deb https://packages.confluent.io/confluent-cli/deb stable main"
+    sudo apt update && sudo apt install confluent-cli
+
+#### YUM (RHEL and CentOS)
+
+Install the latest version of `confluent` to `/usr/bin` (requires `glibc 2.17` or above for `amd64` and `glibc 2.27` or above for `arm64`):
+
+    sudo rpm --import https://packages.confluent.io/confluent-cli/rpm/archive.key
+    sudo yum install yum-utils
+    sudo yum-config-manager --add-repo https://packages.confluent.io/confluent-cli/rpm/confluent-cli.repo
+    sudo yum clean all && sudo yum install confluent-cli
+
+
 #### Scripted installation
```

Comment:
Remove?

Reply:
Windows users might still want this until we release Chocolatey

Reply:
It doesn't work on windows iirc, that's why there's a separate "windows" section underneath it